### PR TITLE
Document Charuco

### PR DESCRIPTION
**Pull request by Juniormunk**

### CODE REVIEW

File: docs/source/docs/calibration/calibration.rst [hunk 1]
```diff
@@ -3,9 +3,9 @@ Calibrating Your Camera
...
 
-.. warning:: While any resolution can be calibrated, resolutions lower than 960x720 are often too low to provide accurate results. Additionally, high resolutions may be too performance intensive for a coprocessor like a Raspberry Pi to handle (solutions to this are being looked into). Thus, we recommend 960x720 when using 3D mode.
+.. warning:: While any resolution can be calibrated, resolutions lower than 960x720 are often too low to provide accurate results. Additionally, high resolutions may be too performance-intensive for a coprocessor like a Raspberry Pi to handle (solutions to this are being looked into). Thus, we recommend 960x720 when using 3D mode.
```
Comment (mcm001) [inline docs/source/docs/calibration/calibration.rst:8]: Is this still accurate with orange pi's?

File: docs/source/docs/calibration/calibration.rst [hunk 2]
```diff
@@ -16,21 +16,21 @@ Accurate camera calibration is required in order to get accurate pose measuremen
 
 1. Practice calibration using your laptop webcam and https://www.calibdb.net/. The target can be found on the website and should be printed out if possible. Once you print it out, try to line up your target with the overlay on the screen as best as possible. The point of this practice is to notice how you are prompted to place targets in certain positions on the screen that make sure you account for all regions of the sensor. The chessboard should (in general) not be facing parallel to the camera (straight on), nor should it be aligned with any of the camera axes (ie, rotated only about an axis going left/right, up/down, or out-of-the-camera).
```
Comment (mcm001) [inline docs/source/docs/calibration/calibration.rst:17]: Maybe warn that calbdb website might not work, but just use as practice?

File: docs/source/docs/calibration/calibration.rst [hunk 3]
```diff
@@ -39,38 +39,35 @@ Following the ideas above should help in getting an accurate calibration.
...
+
+3. Select calibration resolution and fill in the appropriate target data.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. warning:: Ensure that there is no scaling applied during printing (it should be at 100%) and that the PDF is printed as is on regular printer paper. Check the square size with calipers or an accurate measuring device after printing to ensure squares are sized properly, and enter the true size of the square in the UI text box. For optimal results, various resources are available online to calibrate your specific printer if needed.
+We'll next select a resolution to calibrate and populate our pattern spacing, marker size, and board size. The provided chessboard and charuco board are an 8x8 grid of 1 inch square. The provided charuco board uses the 4x4 dictionary with a marker size of 0.75 inches (this board does not need the old OpenCV pattern selector selected). Printers are not perfect you need to measure your calibration target and enter the correct marker size and pattern spacing (aka size of the black square). Finally, once our entered data is correct, we'll click "start calibration."
```
Comment (mcm001) [inline docs/source/docs/calibration/calibration.rst:62]:
```suggestion
We'll next select a resolution to calibrate and populate our pattern spacing, marker size, and board size. The provided chessboard and charuco board are an 8x8 grid of 1 inch square. The provided charuco board uses the 4x4 dictionary with a marker size of 0.75 inches (this board does not need the old OpenCV pattern selector selected). Printers are not perfect, and you need to measure your calibration target and enter the correct marker size and pattern spacing (aka size of the black square) using calipers or similar. Finally, once our entered data is correct, we'll click "start calibration."
```

File: docs/source/docs/calibration/calibration.rst [hunk 4]
```diff
@@ -108,6 +105,13 @@ Calibration images can also be extracted from the downloaded JSON file using `th
    :width: 600
    :alt: Captured calibration images
 
+Calibrating using calibdb
```
Comment (mcm001) [inline docs/source/docs/calibration/calibration.rst:108]: These are the boards they sell right? Not to be confused with the calibration website?

**Review by mcm001 — APPROVED** [review]

File: docs/source/docs/calibration/calibration.rst (hunk 1 above)
Comment (BytingBulldogs3539) [reply docs/source/docs/calibration/calibration.rst:8]: I believe that with RPI 4, 5, and orange pi 5 something like 1280*720ish is a good resolution to recommend I would instead recommend we say that the higher the resolution the worse the performance.

Comment (Juniormunk) [reply docs/source/docs/calibration/calibration.rst:8]: I believe that with RPI 4, 5, and orange pi 5 something like 1280*720ish is a good resolution to recommend I would instead recommend we say that the higher the resolution the worse the performance.

Comment (mcm001) [reply docs/source/docs/calibration/calibration.rst:8]: Yeah, let's avoid naming numbers if we can. "Go forth and do science" or whatever

**Pull request closed without merging**